projects/components/src/select/select.component.ts

```diff
@@ -151,6 +177,7 @@ export class SelectComponent<V> implements AfterContentInit, OnChanges {
 
   public ngAfterContentInit(): void {
     this.selected$ = this.buildObservableOfSelected();
+    this.topControlItems = this.controlItems?.filter(item => item.position === SelectControlOptionPosition.Top);
```

Comment:
nit:  query list can change over time. Can we set this to something like:
```suggestion
    this.topControlItems$ = queryListAndChanges(this.controlItems).pipe(map(items => items.filter(item => item.position === SelectControlOptionPosition.Top)));
```